src/components/MyAccount/Settings/AccountSettingsUtils.ts

```diff
@@ -97,23 +103,39 @@ export const parseAccountSettingsPayload = (
           },
         }
         break
-      case "homeLibraryCode":
-        putRequestPayload.homeLibraryCode = fieldValue
+      case "homeLibrary":
+        putRequestPayload.homeLibraryCode = fieldValue.split("@")[0]
     }
     return putRequestPayload
-  }, {} as SierraPatron)
+  }, {} as PatronUpdateBody)
+}
+
+export const addHomeLibraryObjectToPayload = (homeLibraryTuple, payload) => {
```

Comment:
Looks like this function isn't getting called anywhere. Is it still necessary?

Reply:
removed